front/src/features/books/helperData.tsx

```diff
@@ -0,0 +1,11 @@
+interface InitialState {
+  data: object[];
```

Comment:
zdefiniuj co to za obiekt - object[] nic nie pomaga

Reply:
Done.